.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__
 *.tfvars
 terraform.tfstate
 terraform.tfstate.backup
+*.tfvars
```

Comment:
I don't think `*.tfvars` needs to be listed twice here.

Reply:
🦅👀 Thanks for noticing, fixed in 1e998a0.